cms/tests/test_apphooks.py

```diff
@@ -32,7 +32,7 @@
 APP_MODULE = "cms.test_utils.project.sampleapp.cms_apps"
 
 
-class ApphooksTestCase(CMSTestCase):
+class ApphooksTestCase(ClearURLs, CMSTestCase):
```

Comment:
I'm not 100% sure. I just know we had this issue for quite a while in various forms. We "solved" most of the module/urls caching by having a ridiculous number of root urlconfs and switching between them. But as #4289 shows for some reason that doesn't fix it.